test/strings/search.jl

```diff
@@ -24,41 +24,43 @@ end
 
 # ascii forward search
 for str in [astr, GenericString(astr)]
-    @test_throws BoundsError findnext(isequal('z'), str, 0)
```

Comment:
Can you keep the existing tests unless you're absolutely sure they are redundant? They don't really hurt.

Reply:
Umm, I guess I didn't change any existing test for `ascii forward search`, I added one/two but I guess that's fine. Probably the change it's reflecting is due to a newline being deprecated in the previous line.

Reply:
What I mean is that you'd better add new tests for `findnext(ch, str, i)` (in a clearly separate block) rather than replacing the `findnext(isequal(ch), str, i)` tests. That will also avoid mixing `findfirst(ch, s)` and `findnext(isequal(ch), s, i)` in the same block.

Reply:
Still not addressed.